tests/utils.py

```diff
@@ -169,12 +169,17 @@ def get_image_name_from_csv(image_string, csv_related_images):
     raise ResourceNotFoundError(f"no image with the string {image_string} was found in the csv_dict")
 
 
-def hotplug_resource_and_wait_hotplug_migration_finish(vm, client, sockets=None, memory_guest=None):
-    assert sockets or memory_guest, "No resource for update provided!!!"
-    assert not (sockets and memory_guest), "Hotpluging both cpu and memory at once is not supported!"
```

Comment:
still should be checked, no?

Reply:
hot plugging both cpu and memory at the same time works now, and there are tests that run on upstream kubevirt to verify this.
Unless you think it is not a supported I don't see a reason to keep it.